calico-cloud_versioned_docs/version-19-1/get-started/connect/install-helm.mdx

```diff
@@ -0,0 +1,55 @@
+---
+description: Steps to connect your cluster to Calico Cloud.
+title: Install using Helm
+---
+
+# Connect a cluster to Calico Cloud using Helm
+
+You can connect your cluster to Calico Cloud by generating a unique Helm command in the Calico Cloud Manager UI and running it on your cluster.
+With Helm installations you can also set optional installation parameters to enable certain features.
+
+## Prerequisites
+
+* You have an active Calico Cloud account. You can sign up for a 14-day free trial at [calicocloud.io](https://calicocloud.io).
+* You are signed in to the Calico Cloud Manager UI as a user with the **Owner**, **Admin**, or **DevOps** role.
+* You have at least one cluster that meets our [system requirements](requirements/system-requirements.mdx).
+* You have kubectl access to the cluster.
+* You have installed Helm 3.0 or later on your workstation.
+
+## Install Calico Cloud using Helm
+
+1. From the **Managed Clusters** page, click **Connect Cluster**.
+1. In the **Connect a Cluster** dialog, enter a **Cluster Name** and select a **Cluster Type**.
+1. (Optional) If you must install a specific older release, select the Calico Cloud version you want to install.
+   We always recommend the latest version, which is installed by default.
+1. Click **Advanced Options**, and then select **Install via helm**
+1. Click **Connect** to generate a unique Helm installation command. Copy the command.
+
+   ```bash title="Example of generated Helm installation command"
+   helm repo add calico-cloud https://installer.calicocloud.io/charts --force-update && helm upgrade --install calico-cloud-crds calico-cloud/calico-cloud-crds --namespace calico-cloud --create-namespace && helm upgrade --install calico-cloud calico-cloud/calico-cloud --namespace calico-cloud --set apiKey=ryl34elz8:5kdv6siag:ifk1uwruwlgp7vzn7ecijt5zjbf5p9p1il1ag8877ylwjo4muu19wzg2g8x5qa7x --set installer.clusterName=my-cluster --set installer.calicoCloudVersion=v19.1.0
+   ```
+1. (Optional) To make changes to what features are enabled during installation, paste the command to a text editor and append the `--set` option any of the following key-value pairs.
+
+   | Feature | Key | Values |
+   |---------|-----|--------|
+   | Image Assurance | `installer.components.imageAssurance.state` | `Enabled`, `Disabled` |
+   | Runtime Security | `installer.components.runtimeSecurity.state` | `Enabled`, `Disabled` |
+   | Security Posture Dashboard | `installer.components.securityPosture.state` | `Enabled`, `Disabled` |
+
```

Comment:
It would be good to have the defaults somewhere, which is `Enabled` for Image Assurance and Security Posture.

Runtime Security is bit tricky because the default is `Unmanaged`, which means:
- It will be disabled for new clusters
- It will preserve the runtime security (enabled/disabled) state when upgrading from an older version

Reply:
I've addressed this. Tricky to fit that in, let me know what you think.

Reply:
It looks good to me.